data_steward/gcloud/bq/__init__.py

```diff
@@ -227,20 +229,33 @@ def define_dataset(self, dataset_id: str, description: str,
 
         return dataset
 
-    def copy_dataset(self, input_dataset: str, output_dataset: str) -> list:
+    def copy_dataset(self,
+                     input_dataset: str,
+                     output_dataset: str,
+                     job_config: CopyJobConfig = None) -> list:
         """
         Copies tables from source dataset to a destination datasets
 
         :param input_dataset: fully qualified name of the input(source) dataset
         :param output_dataset: fully qualified name of the output(destination) dataset
+        :param job_config: An optional google.cloud.bigquery.job.CopyJobConfig
         :return: incomplete jobs
         """
+
+        if not job_config:
+            job_config = CopyJobConfig()  # create an empty default
+
```

Comment:
This could be a ternary operator if it is truly an empty default.
```suggestion
        job_config = job_config if job_config else CopyJobConfig()
```

I personally prefer to specify a `WRITE_EMPTY` default value if we are copying
```suggestion
        job_config = job_config if job_config else CopyJobConfig(write_disposition=WriteDisposition.WRITE_EMPTY)
```